src/auth/cephx/CephxKeyServer.cc

```diff
@@ -368,16 +369,25 @@ bool KeyServer::updated_rotating(bufferlist& rotating_bl, version_t& rotating_ve
 {
   Mutex::Locker l(lock);
 
-  _check_rotating_secrets(); 
-
-  if (data.rotating_ver <= rotating_ver)
-    return false;
+  if (data.rotating_ver < rotating_ver)
```

Comment:
shall we check `rotating_ver` against `temp_data.rotating_ver` before setting and returning `rotating_bl`?

also, instead of start a new line with `{`, please put `{` at the end of line.